go/protocol/keybase_v1.go

```diff
@@ -5698,7 +5699,8 @@ type Update struct {
 	Name        string     `codec:"name" json:"name"`
 	Description string     `codec:"description" json:"description"`
 	Type        UpdateType `codec:"type" json:"type"`
-	Asset       Asset      `codec:"asset" json:"asset"`
+	PublishedAt *Time      `codec:"publishedAt,omitempty" json:"publishedAt,omitempty"`
+	Asset       *Asset     `codec:"asset,omitempty" json:"asset,omitempty"`
```

Comment:
Yeah the release tool is updated here to set publishedAt and digest:
https://github.com/keybase/release/pull/1

After merging this PR, we need to merge that right after (before a build is triggered).